src/components/dashboard/Who.js

```diff
@@ -49,6 +51,28 @@ const Who = (props: AmountProps) => {
     [screenProps]
   )
 
+  const canContinue = useCallback(async () => {
+    // =====
+    // await for android keyboard to be closed before redirecting to the next step
+    if (isAndroidNative) {
+      await new Promise(res => {
+        Keyboard.addListener('keyboardDidHide', res)
+        Keyboard.dismiss()
+      })
+    }
+
+    if (isAndroidWeb) {
+      await new Promise(res => {
+        delay(300)
```

Comment:
1. why fixed delay of 300ms ? Isn't exact, we have to check for some event instead. Android could be ran on huge amount of devices, on low-end phones keyboard coud show/hide even more time (about 1sec)
2. is it really works ? delay is just an operator could be used in rxjs chain. it doesn't performs any delays on immediate call.
it could be used as the following:
```javascript
import { of } from 'rxjs'
import { delay } from 'rxjs/operators'

await of(void 0).pipe(delay(300)).asPromise()
```

also you could just use promisified setTimeout:

```
await new Promise(resolve => setTimeout(resolve, 300))
```